src/test/java/life/qbic/openbis/openbisclient/TestOpenBisClient.java

```diff
@@ -0,0 +1,124 @@
+package life.qbic.openbis.openbisclient;
+
+import static org.junit.Assert.assertEquals;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import ch.ethz.sis.openbis.generic.asapi.v3.dto.experiment.ExperimentType;
+import ch.ethz.sis.openbis.generic.asapi.v3.dto.experiment.fetchoptions.ExperimentTypeFetchOptions;
+import ch.ethz.sis.openbis.generic.asapi.v3.dto.property.DataType;
+import ch.ethz.sis.openbis.generic.asapi.v3.dto.property.PropertyAssignment;
+import ch.ethz.sis.openbis.generic.asapi.v3.dto.property.PropertyType;
+import ch.ethz.sis.openbis.generic.asapi.v3.dto.property.fetchoptions.PropertyAssignmentFetchOptions;
+import ch.ethz.sis.openbis.generic.asapi.v3.dto.sample.Sample;
+import ch.ethz.sis.openbis.generic.asapi.v3.dto.sample.fetchoptions.SampleFetchOptions;
+import ch.ethz.sis.openbis.generic.asapi.v3.exceptions.NotFetchedException;
+import life.qbic.openbis.openbisclient.helper.OpenBisClientHelper;
+
+public class TestOpenBisClient {
+
+  IOpenBisClient openbis;
+  OpenBisClientHelper helper;
+
+  @BeforeClass
+  public static void setUpBeforeClass() {}
+
+  @AfterClass
+  public static void tearDownAfterClass() {}
```

Comment:
Why is this included? Is this required to let the tests run?

Reply:
I was not sure, might be an instance of cargo cult programming.

Reply:
In any case they can be used if we ever set up some kind of testing system that needs initialisation and shutdown. Maybe we can remove them until then, though?

Reply:
If you want to keep them its also fine for me :)